addons/binding/org.openhab.binding.enocean/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,46 @@
+Manifest-Version: 1.0
+Automatic-Module-Name: org.openhab.binding.enocean
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: EnOcean Binding
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.enocean;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.4.0.qualifier
+Export-Package: 
+ org.openhab.binding.enocean,
+ org.openhab.binding.enocean.handler,
+ org.openhab.binding.enocean.profiles
+Import-Package: 
+ com.google.common.collect,
+ gnu.io,
```

Comment:
We by now have an ESH serial API (see [here](https://github.com/eclipse/smarthome/tree/master/bundles/io/org.eclipse.smarthome.io.transport.serial/src/main/java/org/eclipse/smarthome/io/transport/serial)), which should be used instead of gnu.io directly (it is a facade, which in the background still uses gnu.io for openHAB).

Reply:
I switched over to the ESH serial API (nice work done here). Using this API I could remove the tcpBridge as this API supports rfc2217. However as long as the NRJavaSerial Bug is not fixed on windows OS, I have to stick with gnu.io. I commented these lines with a TODO. On linux machines the ESH serial API is used. However I get an exception when I try to use rfc2217 (gnu.io.serialport missing). Maybe I have done something wrong with the integration of this API.

Reply:
See my comment below - nrjavaserial should also work on Windows, so no need for gnu.io import anymore.